13 counselor dashboardpage creation

JoyZhang2023 opened the pull request:
UAT video link: https://www.loom.com/share/a6db352796bc42d0867ddce798564146?sid=041cebd4-a7e0-471e-a3e1-21ad06d0940c

This pr resolves #13 

It's a placeholder page for counselor dashboard with nav bar which links to other counselor pages.

https://github.com/user-attachments/assets/31523fa4-6d86-43d8-a029-08529f65ae40

JoyZhang2023 committed:
add dashboard

JoyZhang2023 committed:
add dashboard

JoyZhang2023 committed:
add nav bar

JoyZhang2023 committed:
add nav bar

JoyZhang2023 committed:
fix navbar link

JoyZhang2023 committed:
fix navbar link

JoyZhang2023 committed:
add prop to nav bar

JoyZhang2023 committed:
add prop to nav bar

mx-cifu reviewed (approved):
I was able to locate the specified files easily, and the output when running the program displayed the same results as shown in the UAT video. No bugs became apparent during my test of the branch and the results match the requirements.

nwm516 committed:
scholar's-saga_2_2_home-page-creation (#24)

* Home page creation

Project initialized and home page created

* Added ProfileCard component

Added forgotten ProfileCard component file

JoyZhang2023 committed:
Merge branch '13-counselor-dashboardpage-creation' of https://github.com/LilithLucifer3x6/Scholars-Saga into 13-counselor-dashboardpage-creation
